packages/vscode-extension/package.json

```diff
@@ -139,6 +139,13 @@
           "group": "inline"
         }
       ],
+      "editor/title/run": [
+        {
+          "command": "fx-extension.selectAndDebug",
+          "when": "!inDebugMode && debuggersAvailable && fx-extension.active",
```

Comment:
Should we also check if currently it's teamsfx project? Or it will be shown also when not in teamsfx project, then will this conflict with Java extension?

Reply:
The run button will be shown when the Teams Toolkit is active.

Ps: "teams toolkit is active" = "the project is a teams project" or "user manually open the teams toolkit tree view in other types of project" or "user manually trigger teams toolkit command in other types of project".

As we discussed offline with Jaeyong and Yanliang, in this sprint the run button may be shown in other types of projects. We added some telemetries to track how many users click the run icon under other types of projects and improve the user experience in the next sprint.